src/date-input/interfaces.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { BaseInputProps, InputProps } from '../input/interfaces';
+import { BaseComponentProps } from '../internal/base-component';
+import { FormFieldValidationControlProps } from '../internal/context/form-field-context';
+
+export interface DateInputProps extends BaseInputProps, FormFieldValidationControlProps, BaseComponentProps {
+  /**
+   * Disable value autocompletion when input is blurred.
+   */
+  disableAutocompleteOnBlur?: boolean;
```

Comment:
Could you remind what we are using this property for?

Reply:
Good question. It is usually false and we only really use it in the date-picker: https://github.com/cloudscape-design/components/blob/main/src/date-picker/index.tsx#L118

However, from the code it looks like this usage is not really justified. When the dropdown is open the input blur event can't really happen.

How about I make it false by default and remove from the public API for now?

Reply:
Moving to private works, always a safe bet